src/cnd/cnd.ts

```diff
@@ -329,6 +329,218 @@ export interface SwapProperties {
   [k: string]: any;
 }
 
+/**
+ * The payload returned when fetching one swap on the `/swaps/:id` endpoint
+ */
+export interface SwapResponse extends Entity {
+  properties?: FullSwapProperties;
+}
+
+/**
+ * Element of the array in the payload returned when fetching all swaps on the `/swaps/` endpoint
+ */
+export interface SwapElementResponse extends EmbeddedRepresentationSubEntity {
+  properties?: PartialSwapProperties;
+}
+
+/**
+ * The overall status of a swap
+ */
+export enum SwapStatus {
```

Comment:
One issue I had with the current `Swap` status fields, which is similar to this is that it was too high level. 

I had to parse and process `communication`, `alpha_ledger`, and `beta_ledger` properties to really know 'where' the swap is currently at. What do you think of making these more granular (e.g. `MAKER_FUNDED`, `TAKER_DEPLOYED`, etc.).

Reply:
Ah ok, it's `EscrowStatus`?
Could we consolidate the two statuses?

Reply:
The thing is, `cnd` cannot really know who is the taker and who is the maker in a swap unless you tell it. We only know about cryptographic roles and ledgers but not about who made the initial offer :)

That being said, we can still enrich this status to say more about the individual swap ledgers although the original idea was to keep this really high-level.

Reply:
Hum, not really sure about that. Is it really worth duplicating the information in the response?